OptimizelySDKCore/OptimizelySDKCore/OPTLYDatafileKeys.m

```diff
@@ -60,13 +60,23 @@
 NSString * const OPTLYDatafileKeysVariationKey = @"key";
 NSString * const OPTLYDatafileKeysVariationVariableUsageInstances = @"variables";
```

Comment:
if you want to go this route, and, i think you should, you need to rename OPTLYDatafileKeysVariationVariableUsageInstances to OPTLYDatafileKeysVariationVariables.  Why they changed the name is completely unclear to me.